libraries/RC_Channel/RC_Channel_aux.h

```diff
@@ -91,6 +91,9 @@ class RC_Channel_aux : public RC_Channel {
         k_rcin14                = 64,
         k_rcin15                = 65,
         k_rcin16                = 66,
+        k_ignition              = 67,
```

Comment:
During testing with Paul in which he was adding new values to this Aux_servo_function_t enum, we found that the k_rcin14 and higher would not work outside the simulator because one of the bitmasks involved only had 64 bits.